AK/Platform.h

```diff
@@ -33,7 +33,11 @@
 #ifdef NEVER_INLINE
 #    undef NEVER_INLINE
 #endif
-#define NEVER_INLINE [[gnu::noinline]]
+#if defined(__GNUC__) && !defined(__clang__)
+#    define NEVER_INLINE [[gnu::noinline]]
+#else
+#    define NEVER_INLINE
```

Comment:
there is __attribute__((noinline)), that could work

Reply:
Thank you.